.github/workflows/argilla-sdk.docs.yml

```diff
@@ -7,28 +7,16 @@ concurrency:
 on:
   push:
     paths:
-      - "argilla-sdk/**"
+      - "argilla-sdk/docs/**"
     branches:
       # - "main"
-      - "feat/**"
-      - "gh-pages"
+      # - "develop"
+      - "feat/v2.0.0" # Fixing this branch until we merge everything into develop
```

Comment:
I assumed we would like to include `feat/**` because we might branch off `feat/v2.0.0` into `feat/`-like branches?

Reply:
I've included the `docs/**` pattern if we want to have a specific doc version. Otherwise, we will finish with a lot of intermediate versions.